scripts/build/build_node/Platform/Linux/package-list.ubuntu-bionic.txt

```diff
@@ -13,6 +13,7 @@ libxcb-xinput0                          # For Qt plugins at runtime
 libfontconfig1-dev                      # For Qt plugins at runtime
 libcurl4-openssl-dev                    # For HttpRequestor
 # libsdl2-dev                             # For WWise/Audio
+libssl-dev                              # For system default OpenSSL
```

Comment:
I've added this change in my PR for the Linux upgrade as well: https://github.com/o3de/o3de/pull/8366

Reply:
Yes, I was using this to test in the sandbox environment. This should merge cleanly with your change